docs/ml/vision/detection.md

```diff
@@ -213,7 +213,7 @@ Add the vision service object to the services array in your raw JSON configurati
     "type": "vision",
     "model": "mlmodel",
     "attributes": {
-      "mlmodel_name": "person_detector"
+      "mlmodel_name": "my_vision_service"
```

Comment:
same. but why no <>?

Reply:
The brackets in these examples represent placeholder values for the JSON Template tab, while example strings are used under the JSON Example tab. Compare the two JSON tabs [here](https://docs.viam.com/components/camera/webcam/) for example.

Reply:
ah, got it.